src/pages/Admin/Charity/Banking/useBanking.ts

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from "react";
+import { useTypedWiseMutation } from "services/aws/bankDetails";
+import { useErrorContext } from "contexts/ErrorContext";
+import useDebouncer from "hooks/useDebouncer";
+import { logger } from "helpers";
+import { EMAIL_SUPPORT } from "constants/env";
+import { Currency } from "./CurrencySelector";
+
+const DEFAULT_TARGET_CURRENCY = "USD";
+const ERROR_MSG = `Error loading currencies. Please try again later. If the error persists, please contact ${EMAIL_SUPPORT}.`;
+
+export default function useBanking() {
+  const [targetCurrency, setTargetCurrency] = useState<Currency>();
+  const [expectedFunds, setExpectedFunds] = useState<number>();
+  const [debouncedExpectedFunds] = useDebouncer(expectedFunds, 1000);
```

Comment:
```suggestion
  const [debouncedExpectedFunds, isDebouncing] = useDebouncer(expectedFunds, 1000);
```
`isLoading | isDebouncing` to show loading also when debouncing

Reply:
Will be addressed in PR #2446